### ISSUE TITLE
Error when running marquez-api on Apple M1

### ISSUE THREAD

**Issue by ucg8j** [MarquezProject/marquez#1818]
## Issue
Marquez-api fails on M1.

```log
marquez-api     | INFO  [2021-12-30 13:18:15,093] io.dropwizard.server.ServerFactory: Starting MarquezApp
marquez-api     |     __  ___
marquez-api     |    /  |/  /___ __________ ___  _____  ____
marquez-api     |   / /|_/ / __ `/ ___/ __ `/ / / / _ \/_  /
marquez-api     |  / /  / / /_/ / /  / /_/ / /_/ /  __/ / /_
marquez-api     | /_/  /_/\__,_/_/   \__, /\__,_/\___/ /___/
marquez-api     |                      /_/
marquez-api     |
marquez-api     | java.lang.ExceptionInInitializerError
marquez-api     | 	at org.eclipse.jetty.http.MimeTypes.<clinit>(MimeTypes.java:175)
marquez-api     | 	at org.eclipse.jetty.server.handler.gzip.GzipHandler.<init>(GzipHandler.java:190)
marquez-api     | 	at io.dropwizard.jetty.GzipHandlerFactory.build(GzipHandlerFactory.java:279)
marquez-api     | 	at io.dropwizard.server.AbstractServerFactory.buildGzipHandler(AbstractServerFactory.java:699)
marquez-api     | 	at io.dropwizard.server.DefaultServerFactory.build(DefaultServerFactory.java:175)
marquez-api     | 	at io.dropwizard.cli.ServerCommand.run(ServerCommand.java:49)
marquez-api     | 	at io.dropwizard.cli.EnvironmentCommand.run(EnvironmentCommand.java:60)
marquez-api     | 	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:98)
marquez-api     | 	at io.dropwizard.cli.Cli.run(Cli.java:78)
marquez-api     | 	at io.dropwizard.Application.run(Application.java:94)
marquez-api     | 	at marquez.MarquezApp.main(MarquezApp.java:63)
marquez-api     | Caused by: java.nio.charset.IllegalCharsetNameException: l;charset=iso-8859-1
marquez-api     | 	at java.base/java.nio.charset.Charset.checkName(Unknown Source)
marquez-api     | 	at java.base/java.nio.charset.Charset.lookup2(Unknown Source)
marquez-api     | 	at java.base/java.nio.charset.Charset.lookup(Unknown Source)
marquez-api     | 	at java.base/java.nio.charset.Charset.forName(Unknown Source)
marquez-api     | 	at org.eclipse.jetty.http.MimeTypes$Type.<init>(MimeTypes.java:107)
marquez-api     | 	at org.eclipse.jetty.http.MimeTypes$Type.<clinit>(MimeTypes.java:67)
marquez-api     | 	... 11 more
marquez-api exited with code 1
```

I tried building the image locally:
![image](https://user-images.githubusercontent.com/6220504/147755867-399fe7d7-c153-4ec3-bf71-d07fde5072d9.png)

And without docker buildkit:
```log
$ DOCKER_BUILDKIT=0 docker build - < Dockerfile
Sending build context to Docker daemon   2.56kB
Step 1/22 : FROM eclipse-temurin:11-alpine AS base
11-alpine: Pulling from library/eclipse-temurin
no matching manifest for linux/arm64/v8 in the manifest list entries
```

After a bit of digging I came across [this issue](https://github.com/adoptium/containers/issues/50) - so to get things working I switched the image being used to one that has arm64 compatibility.

## Setup
OSX: 11.2.3
MacBook Pro (13-inch, M1, 2020)
Docker version: 20.10.8

**Comment by wslulciuc** [comment]
Thanks @ucg8j for the detailed issue and opening a PR for the fix!

**Comment by wslulciuc** [comment]
Fixed #1819
